R/target_market_share_company.R

```diff
@@ -11,6 +11,8 @@
 #'   `tmsr_target_weighted_production` and `smsp_target_weighted_production`.
 #' @export
 #'
+#' @family calculate scenario targets
```

Comment:
I'll add "functions to" so this reads better. 

Compare:

![image](https://user-images.githubusercontent.com/5856545/83263580-780b8f80-a184-11ea-93fa-4d581f8ffb49.png)

versus

![image](https://user-images.githubusercontent.com/5856545/83263655-91144080-a184-11ea-9a29-5305271d5355.png)

Reply:
:)